constants.js

```diff
@@ -69,6 +69,10 @@ const SupportedStandards = {
  * @see https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md
  */
 const ERC725YKeys = {
+	LSP0: {
+		// bytes10(keccak256('ExtensionHandler'))
+		ExtensionHandlerPrefix: '0x21c768241ee86a8d564b',
```

Comment:
Can we add the 2 x zero bytes `0x0000` separator at then end. So that people that import this constant don't have to add it themselves.

```suggestion
		ExtensionHandlerPrefix: '0x21c768241ee86a8d564b0000',
```

Like we do for LSP5.

https://github.com/lukso-network/lsp-smart-contracts/blob/d1d19f1e7e4d11558700903002fec67b7cbff47d/constants.js#L104